doc/nginx-configs/blockdaemon-virt-server-config.conf

```diff
@@ -0,0 +1,75 @@
+#
+# Blockdaemon configuration snippet
+#
+# Can be included directly in /etc/nginx/sites-available/ (Debian based distros)
+# or /etc/nginx/conf.d/ (Red Hat based distros)
+#
+# yourserver and RepReplaceMeWithYourAuthToken should be replaced with the
+# appropriate values for your organization
+#
+
+# Blockdaemon with bearer token authentication
+upstream blockdaemon {
+    server yourserver-solana-01.bdnodes.net;
+    server yourserver-solana-02.bdnodes.net;
+    server yourserver-solana-03.bdnodes.net;
+
+    # Maximum 32bit integer value
+    keepalive_requests 2147483647;
+}
+
+server {
+    listen 7900;
+
+    location / {
+        try_files /nonexistent-used-for-try-testing @$http_upgrade;
+    }
+
+    location @websocket {
+        proxy_pass https://blockdaemon$request_uri;
+        proxy_http_version 1.1;
+        proxy_set_header Host $proxy_host;
+        # websocket requirements
+        proxy_set_header Upgrade $http_upgrade;
+        proxy_set_header Connection "Upgrade";
+        # Blockdaemon specific authentication requirement
+        proxy_set_header Authorization "Bearer ReplaceMeWithWithYourAuthToken";
+        proxy_redirect https://$proxy_host http://$server_name:$server_port;
+        proxy_redirect https:// http://;

```

Comment:
`proxy_redirect` is standard-ish boiler plate to apply to `Location` and `Refresh` headers.  The second line I've learned to use in the past because nginx's somewhat inconsistent application of protocol changes on a `proxy_redirect` that includes a host and port.